x/ccv/provider/keeper/throttle.go

```diff
@@ -229,7 +228,9 @@ func (k Keeper) GetAllGlobalSlashEntries(ctx sdktypes.Context) []providertypes.G
 	entries := []providertypes.GlobalSlashEntry{}
 
 	for ; iterator.Valid(); iterator.Next() {
-		recvTime, chainID, ibcSeqNum := providertypes.ParseGlobalSlashEntryKey(iterator.Key())
+		// The below func should not panic, since we should be iterating over keys that're
+		// assumed to be correctly serialized in QueueGlobalSlashEntry.
+		recvTime, chainID, ibcSeqNum := providertypes.MustParseGlobalSlashEntryKey(iterator.Key())
```

Comment:
```suggestion
		// MustParseGlobalSlashEntryKeyshould not panic, since we should be iterating over keys that're
		// assumed to be correctly serialized in QueueGlobalSlashEntry.
		recvTime, chainID, ibcSeqNum := providertypes.MustParseGlobalSlashEntryKey(iterator.Key())
```